webapp/app.py

```
@@ -0,0 +1,138 @@
+        "completed_at": datetime.utcnow().isoformat(),
+        "result": _serialize_result(result),
+        "approvals": approvals,
+        "patches": [_serialize_patch(patch) for patch in patches],
+    }
+
+    append_run(state, run)
+    return jsonify(run), 201
+
+
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", port=8000, debug=True)
```

Ruff (0.14.11)

Possible binding to all interfaces

(S104)